components/eam/src/physics/cam/froude.F90

```diff
@@ -0,0 +1,176 @@
+MODULE froude
```

Comment:
This is the file which should be kept. I didn't realize the other file I commented on was in "components/cam" directory, which should be deleted. Please follow the comments I made to that file to fix issues in this file.

Reply:
Thanks! Requested changes have been applied.